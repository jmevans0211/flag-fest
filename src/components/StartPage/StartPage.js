import React from 'react';
import { Link } from 'react-router-dom';

const StartPage = () => {
  return (
    <div className="start-container">
      <Link to='/flag-fest' >
        <h3 className="start-button" role="button">START GAME</h3>
      </Link>
    </div>
  )

}

export default StartPage;