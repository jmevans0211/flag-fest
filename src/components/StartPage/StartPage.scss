$button-color: #00E1FF;

.start-content {
  text-align: center;
}

.flag-fest-heading {
  font-size: 300px;
  margin-bottom: 10px;
  margin-top: 60px;
}

.start-button-container {
  background-color: $button-color;
  border-radius: 50px;
  margin: auto;
  width: 35%;
}

.start-button {
  color: white;
  font-size: 100px;
  margin-top: 10px;
}