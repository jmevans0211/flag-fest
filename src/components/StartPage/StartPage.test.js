import React from 'react';
import { shallow } from 'enzyme';
import StartPage from './StartPage';

describe ('StartPage', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<StartPage />)
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });



});