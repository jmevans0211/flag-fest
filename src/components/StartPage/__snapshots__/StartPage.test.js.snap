// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartPage should match snapshot 1`] = `
<div
  className="start-content"
>
  <h1
    className="flag-fest-heading"
  >
    Flag Fest
  </h1>
  <Link
    className="router-link"
    to="/flag-fest"
  >
    <div
      className="start-button-container"
    >
      <h3
        className="start-button"
        role="button"
      >
        START GAME
      </h3>
    </div>
  </Link>
</div>
`;
